### DISCUSSION TITLE
Following on social media, Twitter in particular

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#92] [General]
by SCN9A
> We are following 9 accounts on twitter. I think we should follow all major Algorand projects. This is important to build solid presence on social media.

We kind of add handles after we (fully) implement their dApps in ASA Stats or when we get some kind of feedback or help from their side.

That being said, if we get a positive response in any form from [Rand Gallery](https://github.com/asastats/channel/issues/51) or [Algogems](https://github.com/asastats/channel/issues/52), afaic we should add them.

Following ANS could be a thing, but my opinion is that we should wait for NFD on Mainnet and add them together.

**Comment by wxivd** [comment]
> We kind of add handles after we (fully) implement their dApps in ASA Stats or when we get some kind of feedback or help from their side.

> Following ANS could be a thing, but my opinion is that we should wait for NFD on Mainnet and add them together.

I see your point here, If I were ANS and you add me later on, I would ask myself why now?

Other projects may also have some criteria that we do not know about (after reaching a milestone, if we help..etc). 

I am all for a proactive approach. Building bridges and getting information about future updates (features, collaborations..etc) directly from them. 

- Build solid presence on social media
- Build and maintain good working relationship with core projects on Algorand
- Bidirectional approach, more people get to know about ASA Stats
- This also helps inform the MVP mindset

**Reply by ipaleka** [reply]
> > Following ANS could be a thing, but my opinion is that we should wait for NFD on Mainnet and add them together.
> 
> I see your point here, If I were ANS and you add me later on, I would ask myself why now?

For this particular example, the answer is straightforward and transparent as everything else in ASA Stats: "because we don't want to take a side in that battle of yours".
 
> Other projects may also have some criteria that we do not know about (after reaching a milestone, if we help..etc).

Yes. My opinion is that we should not follow ZestBloom until we find out what's with Rand Gallery. And after we get positive feedback from Rand Gallery or after we reach the Beta, then we'll follow ZestBloom anyway.

> I am all for a proactive approach. Building bridges and getting information about future updates (features, collaborations..etc) directly from them.

Twitter following isn't connected with that. That's my point.
 
> * Build solid presence on social media

Our bottleneck is our resources for that, not our following base. There are no obstacles for us to retweet some news from dApp we don't follow besides time and willingness from our side.
 
> * Build and maintain good working relationship with core projects on Algorand

That's not related to Twitter following IMO. Our counterpart to other projects/people act of following somebody on Twitter **has to be** our implementation of their project on the ASA Stats website. Following on Twitter should be just a consequence.

Which are the core projects on Algorand? Why we should follow Pera Wallet if we haven't sold them any of our API yet?
 
> * Bidirectional approach, more people get to know about ASA Stats
> * This also helps inform the MVP mindset

Imo we should not care about that until our API is developed and our marketing campaign is designed and starts. We do not have resources for anything else. This project simply shouldn't lean on your enthusiasm because we don't know where you'd be in two months or so, If you start with a much more active presence then who would take over Twitter if you aren't here? You are not an employee and you won't get your replacement if you burn out and decide to take a break. Only what we can do is to design a marketing campaign that will get funds dedicated to the Twitter activity and then we should be creating tasks based on our needs on Twitter and on the related funds.

Last, but not least: thank you mate for being part of this project! You have brought tremendous improvements and I'm sure you'll continue to bring them in the future too. Also, thank you for this opportunity to me for a sincere discussion.

**Comment by ipaleka** [comment]
I'll use this topic to again emphasize the main driving force of ASA Stats, being a discussion: @SCN9A I'm not defining what is the best for ASA Stats, I'm just taking a stance in a discussion. It's just a side effect that we have got in this case what happens to be my stance, but the only reason for that is because we haven't got a discussion about this so the initial setup has been used.

Everything is a subject to change in ASA Stats under the right conditions - the discussion's outcomes have to reveal a need for a change.

**Comment by ipaleka** [comment]
Please, everybody, provide your thoughts and ideas on how we should take care of our Twitter account administration. Reddit and Discord can live without active administration (in a means of communication), while Twitter can't. 

For now, we can try to add a few more people with administrative privileges (and probably also add a specific "Work on Twitter" task to our rewards system), or we can outsource a specialized company using the admpool funds that will completely take over Twitter.

What are your thoughts?

**Reply by wxivd** [reply]
I don't think we have a shortage of Admins to manage Twitter account. 

We need to provide guidance and scope of practice and there are two main points.

1. Twitter management concerning best practice and guidance 
- Follow / follow back
- Retweet
- Like
- Reply to DMs
- Reply to posts

2. Content we share on Twitter
Projects tend to share different contents, not all are of high quality. 

There is room for discussion here.

A: Can we generate meaningful information from our portfolio tracker? Best marketing approach.
B: Is sharing such information is beneficial to the end user/follower.
C: Is time to generate such data justified?

Type of content:
- Regular graphs or metric shows the growth of user base
- Median utilisation of NFT/ASA ratio among holders
- Median net Algo balance amongst users
- Most link-clicks (AMMs, NFT, AlgoExplorer)
- Infographics to enlighten and share important aspects of portfolio tracking. As Adam Smith put is, educating customers is an integral approach for the success of businesses, specially when there are a lot of options to choose from
- Polls are helpful indeed as long as they do not contain leading questions. Thus it needs thoughtful preparation to help inform our building blocks and engage the community. 

I like to think we hold a position where we should be able to provide some insights about ASA/NFT on Algorand blockchain.

The suggested types of data we share are suitable to be shared and interacted with on Twitter and other social media accounts.

**Reply by ipaleka** [reply]
Feel free to ask for any data I can provide to you programmatically or by checking some insights not available to you directly. This is probably an example of posts and tweets you're looking for: https://www.reddit.com/r/AlgorandOfficial/comments/sl5iws/rand_gallery_overtook_ab2_gallery_in_the_number/ 

If you are able to take care of our Twitter 2 or 3 times a week, like commenting stuff and replying to DMs, I guess it would be best for us to provide funds for you for that job. The same goes with our rewards system, it looks like our best bet is that we simply provide funds for @AlgoRhythMatic on a monthly basis and that way kind of making sure things are done.

**Reply by AlgoRhythMatic** [reply]
@SCN9A - I think you cover a lot great points here! 

> We need to provide guidance and scope of practice

I think it is fair and necessary to ask that the ASA Stats community (all of us here, now in this discussion, and others like it) establish a "style guide" of sorts to dictate what the expectation is in regards to what steps should be taken.

IMO, there are two main areas to delineate:
1) Normal daily/weekly activities that should be performed from now -> advertising phase begins; things you have mentioned above:
     - relationship building with potential API partners
          - watching for inbound DMs from potential new partnerships
          - followup and checkins w/ existing partnerships 
     - general social media persona cultivation
          - tweeting about ourselves in a way that is not pure announcements
          - following interesting projects
          - liking tweets
          - re-tweeting messaging from "friends"

2) Daily/weekly activities that should be performed once we enter the "advertisement" phase: I won't classify bullets here, but my meaning is the phase where ASA Stats incorporates advertising, marketing, and promotional philosophies.  This will take some serious discussion when the time comes, as I think I may have different philosophies than @ipaleka and others, and am more ready to use language that celebrates ASA Stats as being the "<insert_superlative_here>" - subjective self-promotion, along with objective statistics.  Overall point: social media at this phase likely looks a LOT different than it does now, so even more important to establish a style.

> I don't think we have a shortage of Admins to manage Twitter account.

You are correct, we do not have a shortage of Admins that *could manage Twitter (or other social media channels).  The thing to consider: the activity you are describing is typically owned by one or more "Social Media Manager" roles.  Social Media Management is a bit more creative than pure administration, and takes proactive participation - even in our current phase.  Once we have an idea of what daily participation looks like based on discussions here, it will be good to see who is willing to participate at this level - and also establish how it will be rewarded.  Currently our rewards for hourly tasks is maybe lower than it could be for this sort of work.

**Reply by ipaleka** [reply]
During the creation of our introduction video, Mayor Bancelot aka Mayor | CM YouTube provided a kind of template for our videos. It allows us to make announcements or to show a feature or any other information through a consistent visual appearance.

So we can create a series of very short videos (a couple of seconds) and tweet such a video between text tweets.

Also, [ZAMANI](https://discord.com/channels/906917846754418770/917850389930471474/950854669519970327) and [ɢʜᴏꜱᴛ ʀᴇᴀᴄᴏɴ||$ᴄᴀᴘᴀʙʟᴇ](https://discord.com/channels/906917846754418770/917850389930471474/953452355188322304) have applied to help with graphics design.

**Reply by ipaleka** [reply]
> * relationship building with potential API partners

https://twitter.com/asastatscom/status/1532023269464973314

- - - - -

> Opinion: [@asastatscom](https://twitter.com/asastatscom) has the simplest, yet most concise Algorand wallet info. Has every info I need in one UI w/o a click. Kudos 🐊 
> 
> Wish [@PeraAlgoWallet](https://twitter.com/PeraAlgoWallet) models something similar in mobile UI than its current ver. 
> 

We're about to start with the API development this month and the end product will offer all the website's functionalities. Premium API customers will be automatically selected as the DAO governors (owners of the project). For [@PeraAlgoWallet](https://twitter.com/PeraAlgoWallet) we can prepare 5CP + 7WN k8s cluster.

- - - - -

**Reply by Sammyspeed75** [reply]
> 

Good observations @AlgoRhythMatic and @SCN9A, we need to develop a clear style guide for the internal 'social media managers'. More like a consistent brand voice. Then we can go on to flesh out in concise detail, the types of content as mentioned above, and a comprehensive content schedule.

**Reply by AlgoRhythMatic** [reply]
> https://twitter.com/asastatscom/status/1532023269464973314

This would be the Holy Grail so to speak!  Offering some useful integration for official Pera Algo Wallet would be amazing.  As long of a shot as this might seem, it would be an excellent vision to have in mind as to how/where it could fit into some POC tailored to their design aesthetic.

**Reply by AlgoRhythMatic** [reply]
> it looks like our best bet is that we simply provide funds for @AlgoRhythMatic on a monthly basis and that way kind of making sure things are done.

@Ipaleka - I'm happy to continue processing rewards once per month, but have [opened a discussion](https://github.com/asastats/channel/discussions/119) to see if there are other community members or admins that are also interested.

**Reply by ipaleka** [reply]
> > https://twitter.com/asastatscom/status/1532023269464973314
> 
> This would be the Holy Grail so to speak! Offering some useful integration for official Pera Algo Wallet would be amazing. As long of a shot as this might seem, it would be an excellent vision to have in mind as to how/where it could fit into some POC tailored to their design aesthetic.

Well, a mobile wallet app that will have the main ASA Stats functionalities implemented is going to exist in the next year at the latest. We'd be both ready and honored if that happens to be Pera Wallet.

**Comment by ipaleka** [comment]
by kerrilija in Discord DM:

- - - - -

It would be nice to tweet once or twice a week about what ASA Stats is working on right now.

- - - - -
